spec/factories/operations.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :operation, class: CurationConcerns::Operation do
+    operation_type "Test operation"
+
+    trait :failing do
+      status CurationConcerns::Operation::FAILURE
+    end
+
+    trait :pending do
+      status CurationConcerns::Operation::PENDING
+    end
+
+    trait :successful do
+      status CurationConcerns::Operation::SUCCESS
+    end
+
+    factory :batch_create_operation, class: Sufia::BatchCreateOperation do
```

Comment:
@scherztc Here's the factory from your failing test. Is this file not in your env?